content/news/2024-02-03-bevy-0.13/index.md

```diff
@@ -872,6 +872,50 @@ let layout = render_device.create_bind_group_layout(
 );
 ```
 
+## Unload rendering assets from RAM
+
+<div class="release-feature-authors">authors: @JMS55, @mockersf, @brianreavis</div>
+
+Meshes and the textures used to define their materials take up a ton of memory:
+in many games, memory usage is the biggest limitation on the resolution and polygon count of the game!
+Moreover, actually transferring that data from the CPU (and RAM) to the GPU (and VRAM) can be a real performance bottleneck,
```

Comment:
I tend to talk in terms of system ram and VRAM. We haven’t aligned on it, but talking about where the data is is more correct. If VRAM gets full then data can spill into system RAM and tank performance because the GPU has to fetch data from system RAM.